tests/abort/task.yaml

```diff
@@ -0,0 +1,40 @@
+summary: Check task abort
+environment:
+  SNAP_NAME: hello-world
+restore: |
+  echo Resetting snapd state...
+  systemctl stop snapd || true
+  umount /var/lib/snapd/snaps/*.snap 2>&1 || true
+  rm -rf /snap/*
+  rm -rf /var/lib/snapd/*
+  rm -f /etc/systemd/system/snap-*.{mount,service}
+  rm -f /etc/systemd/system/multi-user.target.wants/snap-*.mount
+  systemctl start snapd
+execute: |
+  echo "Abort with invalid id"
+  invalidID="10000000"
+  expected="error: cannot find change with id \"$invalidID\""
+  actual=$(sudo snap abort $invalidID 2>&1) || true
```

Comment:
yes, makes a lot of sense, with the latest changes the FAILED var is checked after each command expected to fail, pls take a look